src/ui/components/MenuItem/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import PropTypes from 'prop-types';
+import { useTheme } from '../../../contexts/ThemeContext';
+
+const useStyles = createUseStyles({
+  root: {
+    borderBottom: '1px solid #ccc',
+    padding: 10,
+    fontSize: 13,
+    cursor: ({ onClick }) => (onClick ? 'pointer' : 'auto'),
+    '&:hover': {
+      backgroundColor: ({ onClick, theme }) => onClick && theme.translucent[10],
+    },
+    '&:last-child': {
+      borderBottom: 'none',
+    },
```

Comment:
If the MenuItem doesn't get an `onClick` prop then we do not give it hover effects. We also do not add a bottom border for the last menu item.